modules/compose/compose_api_test.go

```diff
@@ -141,14 +141,15 @@ func TestDockerComposeAPIWithProfiles(t *testing.T) {
 
 	for name, test := range testcases {
 		t.Run(name, func(t *testing.T) {
+			var compose ComposeStack
```

Comment:
question: is this and the change to cleanup needed?

Reply:
No, neither of those commits / changes are absolutely necessary, but I though it was a great place to practice the "accept interfaces" recommendation in the blog which you referenced.

I'm happy to remove / revert the changes / commits though.  

In that case, we may want to consider adding the interface implementation check which I recommended.  Otherwise, I could see future changes on the `ComposeStack` interface breaking the condition that `DockerCompose` satisfies said interface.